variants/play/6.4.3.0/variables.json

```diff
@@ -0,0 +1,22 @@
+{
+  "shopware": {
+    "version": "6.4.3.0",
+    "download_url": "https://www.shopware.com/en/Download/redirect/version/sw6/file/install_v6.4.2.1_eccc23da5632314390e2a3715afb72ab0a6d5ab9.zip"
```

Comment:
pls change download link to https://www.shopware.com/de/Download/redirect/version/sw6/file/install_v6.4.3.0_1783a8a29c331b9568274c727b3dae5f73ba142a.zip

Reply:
my bad, fixed!